05-mvc-with-rails/rails-sessions-and-auth-with-devise/readme.md

```diff
@@ -0,0 +1,365 @@
+# Devise! and sessions
+
+## Learning Objectives
+- explain what state is in a web application
+- explain how sessions give state to a web application
+- explain how user authentication utilizes sessions
+- Define and then access a session variable in a Rails application.
+- set session hash key value pairs inside of a rails application
+- implement user authentication into a web application utilizing the devise gem
+- implement useful helper methods devise provides
```

Comment:
friendly reminder to update https://github.com/ga-dc/curriculum/blob/master/05-mvc-with-rails/readme.md if you haven't already